manager/controllers/motion/batchtransfer_controller_test.go

```diff
@@ -52,7 +54,15 @@ var _ = Describe("BatchTransfer Controller", func() {
 			batchTransfer := &motionv1.BatchTransfer{}
 			err = yaml.Unmarshal(batchTransferYAML, batchTransfer)
 			Expect(err).ToNot(HaveOccurred())
-
+			registry := os.Getenv("DOCKER_HOSTNAME")
+			registryNamespace := os.Getenv("DOCKER_NAMESPACE")
+			if len(registry) > 0 && registry != "ghcr.io" {
```

Comment:
There is a new environment variable for the tag now in master branch `DOCKER_TAGNAME`. Can you please also use this one to make sure we won't get any surprises in future?